### ISSUE TITLE
Category on events intergration

### ISSUE THREAD

**Issue by zdanise** [COS301-SE-2024/WhatsOn-UP#143]

**Comment by zdanise** [comment]
@NokukhanyaNdlovu I am getting a bad request is the metadata changed to List<String> ?

**Comment by NokukhanyaNdlovu** [comment]
no, it hasn't been changed in the backend. I was hoping to make all Category related pages once all backend(database) changes have been "finalized" and completed. But in the meantime, if you look in EventService.dart under fetchUniqueCategories function, you will see how i stored  the catergories as a List String. ( this is just a temporary solution for now)
